source/manual/local-frontend-development.html.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Local frontend development
+parent: "/manual.html"
+layout: manual_layout
+section: Frontend
+type: learn
+owner_slack: "#govuk-2ndline"
+last_reviewed_on: 2019-07-08
+review_in: 6 months
+---
+
+Make the following changes to your local setup:
+
+In [govuk_app_config], add your local domain to the allowed CORS domains:
+
+```
+  GOVUK_DOMAINS = [
+    '*.publishing.service.gov.uk',
+    # "*.#{ENV['GOVUK_APP_DOMAIN_EXTERNAL'] || ENV['GOVUK_APP_DOMAIN'] || 'dev.gov.uk'}"
+     '*.dev.gov.uk'
+  ].uniq.freeze
+```
+
+You need to change the frontend app you are working on to use the modified [govuk_app_config]. For example in [frontend] update the Gemfile:
```

Comment:
Again, this is only relevant when trying to run a frontend app against local static. Perhaps we could split this guide into sections? E.g:

1. Running a frontend app against live: `./startup.sh --live`
2. Running a frontend app against local changes in govuk_publishing_components
3. Running a frontend app against local static